app/assets/javascripts/admin/addon/components/admin-config-areas/components.gjs

```diff
@@ -265,6 +266,16 @@ class ComponentRow extends Component {
     }
   }
 
+  get description() {
+    const remoteUrl = this.args.component.remote_theme?.remote_url;
+    if (!remoteUrl) {
+      return;
+    }
+    return (
+      this.args.component.description ?? descriptionForRemoteUrl(remoteUrl)
+    );
```

Comment:
```suggestion
    return (
      this.args.component.description ?? (remoteUrl && descriptionForRemoteUrl(remoteUrl))
    );
```

If there is no `remoteUrl`, I think that we should still try to use component.description

Reply:
Good catch, fixed!